staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -3000,6 +3001,15 @@ func (d *NodeDescriber) Describe(namespace, name string, describerSettings descr
 		return "", err
 	}
 
+	lease, err := d.CoordinationV1().Leases(corev1.NamespaceNodeLease).Get(name, metav1.GetOptions{})
```

Comment:
I remember there was a feature gate. Have we removed that? If not, maybe still guard this behind the feature gate?

Reply:
I don't think it's needed - we're going to GA it in 1.17 anyway and with GA the process is to just leave the feature gate itself but remove all checks on it.

So I'm going to leave as is. Note that if the feature is disabled, the only consequence is that we will write, e.g. "holder identity: <unset>", so I don't think it's actuallu a big deal.